bucket/sonar-scanner.json

```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/",
+    "description": "Scan code for Sonarqube when there is no specific scanner for your build system.",
```

Comment:
This description do not provide sufficient information.

This application is some sort of source code analyzer right?

Reply:
I've updated the description which will hopefully be clearer. However nobody would just download this, they would already be familiar with _SonarQube_ as you need to connect the _sonar-scanner_ to a _SonarQube_ instance (local, self hosted or SaaS).